### PR TITLE
Updating plugins for 1.3

### PR DESCRIPTION
**Pull request by jesuino**
Opening PR to start the work and the export for Backstage plugins in 1.3

For this version I also excluded plugins that are included in RHDH as wrappers, the recommendation is to use the existing wrapper.

### CODE REVIEW

File: plugins-list.yaml [hunk 1]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/auth-backend-module-oauth2-proxy-provider:
-plugins/auth-backend-module-oidc-provider:
+# plugins/auth-backend-module-oidc-provider: # Included as static plugin in RHDH
 plugins/auth-backend-module-okta-provider:
+plugins/auth-backend-module-onelogin-provider:
 plugins/auth-backend-module-pinniped-provider:
 plugins/auth-backend-module-vmware-cloud-provider:
-#plugins/catalog: ==> Added as a static plugin in RHDH
-#plugins/catalog-backend: ==> Added as a static plugin in RHDH
+plugins/auth-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: This is not a plugin, but a `-node` package which will be embedded in the `auth-backend` plugin.

File: plugins-list.yaml [hunk 2]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/auth-react:
```
Comment (davidfestal) [inline plugins-list.yaml]: Same as above here, but for frontend plugins.

File: plugins-list.yaml [hunk 3]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/bitbucket-cloud-common:
```
Comment (davidfestal) [inline plugins-list.yaml]: same as above, but for both backend and frontend plugins.

File: plugins-list.yaml [hunk 4]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+# plugins/catalog  ==> Added as a static plugin in RHDH
+# plugins/catalog-backend  ==> Added as a static plugin in RHDH
 plugins/catalog-backend-module-aws:
 plugins/catalog-backend-module-azure:
 plugins/catalog-backend-module-backstage-openapi:
-plugins/catalog-backend-module-bitbucket-cloud:
-plugins/catalog-backend-module-bitbucket-server:
+plugins/catalog-backend-module-bitbucket:
+# plugins/catalog-backend-module-bitbucket-cloud  ==> wrapper
```
Comment (davidfestal) [inline plugins-list.yaml]:
```suggestion
# plugins/catalog-backend-module-bitbucket-cloud  ==> already embedded in RHDH
```

File: plugins-list.yaml [hunk 5]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+# plugins/catalog-backend-module-ldap  ==> wrapper
+# plugins/catalog-backend-module-logs  ==> wrapper
+# plugins/catalog-backend-module-msgraph  ==> wrapper
 plugins/catalog-backend-module-openapi:
 plugins/catalog-backend-module-puppetdb:
 plugins/catalog-backend-module-scaffolder-entity-model:
 plugins/catalog-backend-module-unprocessed:
-#plugins/catalog-graph: ==> Added as a static plugin in RHDH 
-#plugins/catalog-import: ==> Added as a static plugin in RHDH
+plugins/catalog-common:
```
Comment (davidfestal) [inline plugins-list.yaml]: That's a common package, not a plugin.

File: plugins-list.yaml [hunk 6]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+# plugins/catalog-graph ==> Added as a static plugin in RHDH :
+# plugins/catalog-import ==> Added as a static plugin in RHDH
+plugins/catalog-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: Node package, not a plugin

File: plugins-list.yaml [hunk 7]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/catalog-react:
```
Comment (davidfestal) [inline plugins-list.yaml]: React package, not a plugin

File: plugins-list.yaml [hunk 8]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/catalog-unprocessed-entities:
+plugins/catalog-unprocessed-entities-common:
```
Comment (davidfestal) [inline plugins-list.yaml]: Common package, not a plugin

File: plugins-list.yaml [hunk 9]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/config-schema:
 plugins/devtools:
 plugins/devtools-backend:
+plugins/devtools-common:
```
Comment (davidfestal) [inline plugins-list.yaml]: Common package, not a plugin

File: plugins-list.yaml [hunk 10]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/events-backend-module-aws-sqs:
 plugins/events-backend-module-azure:
 plugins/events-backend-module-bitbucket-cloud:
 plugins/events-backend-module-gerrit:
-#plugins/events-backend-module-github: # ==> the new backend system is not exported as default export (Error: Backend plugin is not valid for dynamic loading: it should either export a BackendFeature or BackendFeatureFactory as default export, or export a const dynamicPluginInstaller: BackendDynamicPluginInstaller field as dynamic loading entrypoint)
-#plugins/events-backend-module-gitlab: # ==> the new backend system is not exported as default export (Error: Backend plugin is not valid for dynamic loading: it should either export a BackendFeature or BackendFeatureFactory as default export, or export a const dynamicPluginInstaller: BackendDynamicPluginInstaller field as dynamic loading entrypoint)
+# plugins/events-backend-module-github ==> Added as a static plugin in RHDH
+# plugins/events-backend-module-gitlab ==> Added as a static plugin in RHDH
+plugins/events-backend-test-utils:
+plugins/events-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: Node package, not a plugin

File: plugins-list.yaml [hunk 11]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/example-todo-list:
 plugins/example-todo-list-backend:
+plugins/example-todo-list-common:
```
Comment (davidfestal) [inline plugins-list.yaml]: Common package, not a plugin

File: plugins-list.yaml [hunk 12]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/home:
-plugins/kubernetes:
-plugins/kubernetes-backend:
+plugins/home-react:
```
Comment (davidfestal) [inline plugins-list.yaml]: React package, not a plugin

File: plugins-list.yaml [hunk 13]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+# plugins/kubernetes  ==> wrapper
+# plugins/kubernetes-backend  ==> wrapper
+plugins/kubernetes-cluster:
```
Comment (davidfestal) [inline plugins-list.yaml:64]: Not a plugin, afaict

File: plugins-list.yaml [hunk 14]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/kubernetes-common:
```
Comment (davidfestal) [inline plugins-list.yaml]: Common package, not a plugin

File: plugins-list.yaml [hunk 15]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/kubernetes-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: Node package, not a plugin

File: plugins-list.yaml [hunk 16]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/kubernetes-react:
```
Comment (davidfestal) [inline plugins-list.yaml]: React package, not a plugin

File: plugins-list.yaml [hunk 17]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/notifications:
-plugins/notifications-backend: --embed-package @backstage/plugin-signals-node
+plugins/notifications-backend:
+plugins/notifications-backend-module-email:
```
Comment (davidfestal) [inline plugins-list.yaml]:
```suggestion
plugins/notifications-backend-module-email: --embed-package @backstage/plugin-notifications-node --embed-package @backstage/plugin-signals-node
```

File: plugins-list.yaml [hunk 18]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/notifications-common:
```
Comment (davidfestal) [inline plugins-list.yaml]: Common package, not a plugin

File: plugins-list.yaml [hunk 19]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/notifications-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: Node package, not a plugin

File: plugins-list.yaml [hunk 20]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/org:
-#plugins/permission-backend: ==> Added as a static plugin in RHDH
-#plugins/permission-backend-module-policy-allow-all: ==> Not meaningful for RHDH
+plugins/org-react:
```
Comment (davidfestal) [inline plugins-list.yaml]: React package, not a plugin

File: plugins-list.yaml [hunk 21]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+# plugins/permission-backend ==> Added as a static plugin in RHDH
+# plugins/permission-backend-module-policy-allow-all ==> Added as a static plugin in RHDH
+plugins/permission-common:
```
Comment (davidfestal) [inline plugins-list.yaml]: Common package, not a plugin

File: plugins-list.yaml [hunk 22]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/permission-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: Node package, not a plugin

File: plugins-list.yaml [hunk 23]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/permission-react:
```
Comment (davidfestal) [inline plugins-list.yaml]: React package, not a plugin

File: plugins-list.yaml [hunk 24]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/proxy-backend:
```
Comment (davidfestal) [inline plugins-list.yaml]: Isn't it in the RHDH as a static plugin ?

File: plugins-list.yaml [hunk 25]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
-plugins/scaffolder-backend-module-gitlab:
+# plugins/scaffolder-backend-module-github  ==> wrapper
+# plugins/scaffolder-backend-module-gitlab  ==> wrapper
+plugins/scaffolder-backend-module-notifications:
 plugins/scaffolder-backend-module-rails:
 plugins/scaffolder-backend-module-sentry:
 plugins/scaffolder-backend-module-yeoman:
-#plugins/search: ==> Added as a static plugin in RHDH
-#plugins/search-backend: ==> Added as a static plugin in RHDH
+plugins/scaffolder-common:
```
Comment (davidfestal) [inline plugins-list.yaml]: Common package, not a plugin

File: plugins-list.yaml [hunk 26]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/scaffolder-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: Node package, not a plugin

File: plugins-list.yaml [hunk 27]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/scaffolder-node-test-utils:
```
Comment (davidfestal) [inline plugins-list.yaml]: Not a plugin, but a util package

File: plugins-list.yaml [hunk 28]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/scaffolder-react:
```
Comment (davidfestal) [inline plugins-list.yaml]: React package, not a plugin

File: plugins-list.yaml [hunk 29]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+# plugins/search  ==> Added as a static plugin in RHDH
+# plugins/search-backend  ==> Added as a static plugin in RHDH
 plugins/search-backend-module-catalog:
```
Comment (davidfestal) [inline plugins-list.yaml]: Some of these search modules are probably included in the RHDH already aren't they ?

File: plugins-list.yaml [hunk 30]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/search-backend-module-elasticsearch:
 plugins/search-backend-module-explore:
 plugins/search-backend-module-pg:
 plugins/search-backend-module-stack-overflow-collator:
 plugins/search-backend-module-techdocs:
+plugins/search-backend-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: Node package, not a plugin

File: plugins-list.yaml [hunk 31]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/search-common:
```
Comment (davidfestal) [inline plugins-list.yaml]: Node package, not a plugin

File: plugins-list.yaml [hunk 32]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/search-react:
```
Comment (davidfestal) [inline plugins-list.yaml]: React package, not a plugin

File: plugins-list.yaml [hunk 33]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/signals:
 plugins/signals-backend:
-plugins/techdocs:
-#plugins/techdocs-backend: ==> Error: Dynamic plugins do not support native plugins. the following native modules have been transitively detected: - cpu-features
+plugins/signals-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: Node package, not a plugin

File: plugins-list.yaml [hunk 34]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/signals-react:
```
Comment (davidfestal) [inline plugins-list.yaml]: React package, not a plugin

File: plugins-list.yaml [hunk 35]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+# plugins/techdocs  ==> wrapper
+plugins/techdocs-addons-test-utils:
```
Comment (davidfestal) [inline plugins-list.yaml]: Not a plugin, but just a utility package.

File: plugins-list.yaml [hunk 36]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+# plugins/techdocs-backend  ==> wrapper
 plugins/techdocs-module-addons-contrib:
```
Comment (davidfestal) [inline plugins-list.yaml:101]: We should check whether it is. plugin (see the `backstage.role` value in the `package.json`.

File: plugins-list.yaml [hunk 37]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/techdocs-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: Node package, not a plugin

File: plugins-list.yaml [hunk 38]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
+plugins/techdocs-react:
```
Comment (davidfestal) [inline plugins-list.yaml]: React package, not a plugin

File: plugins-list.yaml [hunk 39]
```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
...
 plugins/user-settings:
 plugins/user-settings-backend:
+plugins/user-settings-common
```
Comment (davidfestal) [inline plugins-list.yaml]: Common package, not a plugin

**Review by davidfestal — CHANGES_REQUESTED** [review]

File: plugins-list.yaml (hunk 7 above)
Comment (jesuino) [reply plugins-list.yaml]: EDIT - reminded to check it in backstage and removed it

File: plugins-list.yaml (hunk 10 above)
Comment (jesuino) [reply plugins-list.yaml]: I removed test-utils. Are you also talking about about `events-node`?

File: plugins-list.yaml (hunk 24 above)
Comment (jesuino) [reply plugins-list.yaml]: I see [corporate-proxy.ts](https://github.com/janus-idp/backstage-showcase/blob/release-1.3/packages/backend/src/corporate-proxy.ts) in show case, is this the replacement for this plugin?

File: plugins-list.yaml (hunk 29 above)
Comment (jesuino) [reply plugins-list.yaml]: I could not find it in wrappers, where else can I check?

File: plugins-list.yaml (hunk 36 above)
Comment (jesuino) [reply plugins-list.yaml:101]: It is listed as `frontend-plugin-module`

File: plugins-list.yaml [hunk 40]
```diff
@@ -1,10 +1,11 @@
 plugins/api-docs:
 plugins/api-docs-module-protoc-gen-doc:
 plugins/app-backend:
+plugins/app-node:
```
Comment (davidfestal) [inline plugins-list.yaml]: Not a plugin

File: plugins-list.yaml [hunk 41]
```diff
@@ -16,31 +17,37 @@ plugins/auth-backend-module-guest-provider:
 plugins/auth-backend-module-microsoft-provider:
 plugins/auth-backend-module-oauth2-provider:
 plugins/auth-backend-module-oauth2-proxy-provider:
-plugins/auth-backend-module-oidc-provider:
+# plugins/auth-backend-module-oidc-provider: # Included as static plugin in RHDH
```
Comment (davidfestal) [inline plugins-list.yaml]: Why is the text different here ? Compare with `Added as a static plugin in RHDH`

File: plugins-list.yaml (hunk 24 above)
Comment (davidfestal) [reply plugins-list.yaml]: no, I don't think so

File: plugins-list.yaml (hunk 29 above)
Comment (davidfestal) [reply plugins-list.yaml]: in backend/index.ts, where static plugins are wired

File: plugins-list.yaml (hunk 36 above)
Comment (davidfestal) [reply plugins-list.yaml:101]: OK.
Let's keep it.

File: plugins-list.yaml (hunk 10 above)
Comment (davidfestal) [reply plugins-list.yaml]: yes.

File: plugins-list.yaml (hunk 13 above)
Comment (jesuino) [reply plugins-list.yaml:64]: `plugins/kubernetes-cluster` has the role `frontend-plugin`

File: plugins-list.yaml (hunk 24 above)
Comment (jesuino) [reply plugins-list.yaml]: ok, thanks, I see it [here](https://github.com/janus-idp/backstage-showcase/blob/release-1.3/packages/backend/src/index.ts#L83), removing from the list

File: plugins-list.yaml [hunk 42]
```diff
@@ -16,31 +16,35 @@ plugins/auth-backend-module-guest-provider:
...
+# plugins/catalog-backend-module-ldap  ==> already embedded in RHDH
+# plugins/catalog-backend-module-logs  ==> already embedded in RHDH
+# plugins/catalog-backend-module-msgraph  ==> already embedded in RHDH
 plugins/catalog-backend-module-openapi:
 plugins/catalog-backend-module-puppetdb:
 plugins/catalog-backend-module-scaffolder-entity-model:
 plugins/catalog-backend-module-unprocessed:
-#plugins/catalog-graph: ==> Added as a static plugin in RHDH 
-#plugins/catalog-import: ==> Added as a static plugin in RHDH
+# plugins/catalog-graph ==> Added as a static plugin in RHDH :
```
Comment (davidfestal) [inline plugins-list.yaml]:
```suggestion
# plugins/catalog-graph ==> Added as a static plugin in RHDH
```

File: plugins-list.yaml [hunk 43]
```diff
@@ -16,31 +16,35 @@ plugins/auth-backend-module-guest-provider:
 plugins/auth-backend-module-microsoft-provider:
 plugins/auth-backend-module-oauth2-provider:
 plugins/auth-backend-module-oauth2-proxy-provider:
-plugins/auth-backend-module-oidc-provider:
+# plugins/auth-backend-module-oidc-provider: # Added as static plugin in RHDH
```
Comment (davidfestal) [inline plugins-list.yaml]:
```suggestion
# plugins/auth-backend-module-oidc-provide ==> Added as static plugin in RHDH
```

**Review by davidfestal — CHANGES_REQUESTED** [review]

**Comment by davidfestal** [comment]
Another question: did you plan to test a number of frontend plugins inside RHDH, in order to also allow providing the appropriate `app-config.dynamic.yaml` ?

File: plugins-list.yaml (hunk 42 above)
Comment (jesuino) [reply plugins-list.yaml]: done, thanks

File: plugins-list.yaml (hunk 43 above)
Comment (jesuino) [reply plugins-list.yaml]: done

**Review by davidfestal — APPROVED** [review]

**Pull request merged**